packages/neuron-wallet/src/controllers/app/menu.ts

```diff
@@ -361,6 +362,18 @@ const updateApplicationMenu = (mainWindow: BrowserWindow | null) => {
           loadTransaction(URL.OfflineSign, json, filePath)
         },
       },
+      {
+        label: t('application-menu.tools.broadcast-transaction'),
+        enabled: hasCurrentWallet,
```

Comment:
`Broadcast transaction` doesn't reply on `which wallet is active` because it doesn't require an action of signing transactions